Dockerfile

```diff
@@ -42,8 +42,11 @@ COPY . /build
 RUN cargo build --release
 
 # test
-RUN cargo test --release -p mashnet-node-runtime
-
+RUN cargo test --release -p ctype
+RUN cargo test --release -p delegation
+RUN cargo test --release -p did
+RUN cargo test --release -p error
+RUN cargo test --release -p portablegabi
```

Comment:
```suggestion
RUN cargo test --all
```

Reply:
We should still use the release version, since it was already built in the previous step